src/entities/Peoples.js

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+
+import { gql } from "apollo-boost";
+import { Query } from "react-apollo";
+import { useParams } from "react-router";
+
+
+import Paginator from '../Paginator.js'
+
+import './customcards.css'
+
+import { Button, Container, Table} from 'react-bulma-components';
+
+const Q = gql`
+  query AllPersons($first: Int, $skip: Int) {
+    allPersons(first: $first, skip: $skip) {
+      id
+      name
+      birthYear
+      gender
+      height
+    }
+  }
+`;
+
+
+const perPage = 10;
+const checkOffset = 5;
+
+const Peoples = () => {
+  let { page } = useParams();
+
+  if(page===null){
+    page = 1;
+  }
+
+  let isAbleToNext = null;
+  let isAbleToPrevious = (page > 1);
+
+  return (
+    <Query
+      query={Q}
+      variables={{
+        first: perPage + checkOffset,
+        skip: perPage * (page - 1)
+      }}
+      fetchPolicy="cache-and-network"
+    >
+      {({ loading, error, data, fetchMore }) => {
+        if (loading) return <Button loading={true} color='black'>Please Wait</Button> ;
+        if (error) return `Error! ${error.message}`;
+        return (
+          <Container>
+            <Table striped={false}>
```

Comment:
Having the whole UI in one component is a bad practice, try dividing the component into basic units. For eg, you can have one child component that just renders a row based on the props it receives. Try to make simpler components.

Reply:
So i should create models of the query and the table columns, right?

Reply:
Right now, you have the whole table together as one component, what you should do instead is write a simple function component that returns just a table row. This would be your child component. In you parent component, you can have the table and the table header (columns),which will keep calling the child component for each row.